src/external/governance/IGovernor.sol

```diff
@@ -25,25 +21,25 @@ interface IGovernanceContract {
     // Errors
 
     /// @notice The given address is invalid
-    error InvalidAddress(address adr);
+    error Governance__InvalidAddress(address adr);
```

Comment:
I assume you've changed these before changing the name of the contract :D They all have governance instead of governor

```suggestion
    error Governor__InvalidAddress(address adr);
```

Reply:
This comment is meant for all the errors